release-team/role-handbooks/release-signal/README.md

```diff
@@ -137,18 +138,39 @@ stabilization phase.
   the [teams.yaml](https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-release/teams.yaml).
   _Coordinate with the release lead to make multiple changes to this file in one PR._
 - Plan release support and status reporting. Setup [CI-Signal project board](/release-team/role-handbooks/release-signal/project-boards.md#setting-up-the-ci-signal-project-board) and the [Bug Triage Project Board](/release-team/role-handbooks/release-signal/project-boards.md#setting-up-the-bug-triage-project-board).
-- Create an excel sheet to track various assignments during the release cycle. [Example](https://docs.google.com/spreadsheets/d/1lYbCOGvG9jD-AVT0Ajc3LoSLNKHC6xIIy1t6BJPlO7E/edit?usp=sharing)
 - Release Signal Shadows must be a member of the Kubernetes organization and, therefore, apply for membership by opening an issue
   on [kubernetes/org](https://github.com/kubernetes/org) (
   see [Issue template](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.yml&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E)).
 - Organizing an onboarding meeting with shadows to walk through this handbook and useful tools like [Testgrid](https://github.com/GoogleCloudPlatform/testgrid),
   [Triage](https://github.com/kubernetes/test-infra/tree/master/triage), [Spyglass](https://docs.prow.k8s.io/docs/spyglass/), [Prow](https://docs.prow.k8s.io/docs/), and [Tide](https://docs.prow.k8s.io/docs/components/core/tide/).
 
+#### Create Assignment Excel Sheet
+
+Create an excel sheet to track various assignments during the release cycle. [Example](https://docs.google.com/spreadsheets/d/1lYbCOGvG9jD-AVT0Ajc3LoSLNKHC6xIIy1t6BJPlO7E/edit?usp=sharing)
```

Comment:
Can we also add the example excel sheet to github in case this google link is not available one day?

Reply:
Sorry for the quick merge.
I thought that `approve` without commenting `/lgtm` will not trigger adding the `lgtm` label.

Reply:
The assignment excel sheet example is very helpful.

Should we also link it in the weekly RT meeting doc (under Release Signal team updates) as well?

Reply:
> Can we also add the example excel sheet to github in case this google link is not available one day?

Sure, I can add it to the GH repo.

> Should we also link it in the weekly RT meeting doc (under Release Signal team updates) as well?

This sheet is mainly internal assignments, so I don't see much value in adding this to the updates doc. I'll add the link if you think it provides some value.

Reply:
>This sheet is mainly internal assignments, so I don't see much value in adding this to the updates doc. I'll add the link if you think it provides some value.

Public visibility to Release leads & other community stakeholders on Release Signal activities.